.github/workflows/docker.yml

```diff
@@ -0,0 +1,63 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: Build and Push PDO Docker Images
+
+on:
+  workflow_dispatch:
+
+  pull_request:
+    types: [closed]
```

Comment:
Before seeing below if, i stumbled over this [closed] and the lack of something like a type [merged].  Googling a bit i found  https://github.com/orgs/community/discussions/26724#discussioncomment-3253093 which proposes an arguably simpler solution to achieve the same effect: `on: push: branches: - main `

Reply:
Actually 

1. The comment at the top of that issue is actually wrong (you can detect a merged PR)
2. Even in that series of responses, the conclusion is to use the pull_request event with a check for merged

I'm going to leave this as is.